sejima-library/app/src/main/java/com/rlab/sejima/features/MUTextField.java

```diff
@@ -329,17 +328,15 @@ public int getAlignment() {
      * </ul>
      */
     public void setAlignment(int alignment) {
-        RelativeLayout.LayoutParams ll = (LayoutParams) mTVLabel.getLayoutParams();
-        ll.removeRule(mAlignment);
-        ll.addRule(alignment, RelativeLayout.TRUE);
-        mTVLabel.setLayoutParams(ll);
-
         if(Gravity.END == alignment){
```

Comment:
It looks like `mETInput.setGravity(alignment); mTVLabel.setGravity(alignment);` do the same thing?